CleanProFinder/CleanProFinder.Mobile/Views/CustomerStartingPage.xaml.cs

```diff
@@ -0,0 +1,12 @@
+using CleanProFinder.Mobile.ViewModels;
+
+namespace CleanProFinder.Mobile.Views;
+
+public partial class CustomerStartingPage : ContentPage
+{
+	public CustomerStartingPage(CustomerStartingViewModel viewModel)
+	{
+        InitializeComponent();
```

Comment:
Add indentation in CustomerStartingPage constructor

Reply:
Resolved